src/routes/learn/getting-started-with-solid/building-ui-with-components.mdx

```diff
@@ -0,0 +1,109 @@
+<title>Building User Interfaces with Components</title>
+
+## What are components?
+
+Components are functions that define how your application will look. Each component generally represents a small part of your application's entire user interface. Splitting your application into components has many advantages: components are small, reusable, easily-tested, and enable different team members to work on different parts of the application without stepping on each other.
```

Comment:
```suggestion
Components are functions that describe how your application will look. Each component generally represents a small part of your application's user interface. Splitting your application into components has many advantages: components are small, reusable, and easily testable. They allow different team members to work on different parts of the application without stepping on each other's toes.
```

Reply:
Just trying to have smaller and simpler sentences in general. More non-techy terms `describe` instead of `define`. Not entirely sure the best approach but as a reader its always easier to read.